src/walk_dirs.rs

```diff
@@ -119,9 +118,12 @@ pub fn discover_fences_and_files(start_path: &str) -> Vec<WalkFileData> {
                   let _working_dir_path: &Path = &WORKING_DIR_PATH;
                   let source_file_path = RelativePath::from_path(&file_path);
 
+                  let imports = get_imports_from_file(&file_path);
+                  let imports_map = get_imports_map(&imports, &file_path);
```

Comment:
These functions are only ever called in succession -- I think they should be made private and instead the module should only export `get_imports_map_from_file`.